distributed_training/utils/config.py

```diff
@@ -214,7 +200,7 @@ def add_args(cls, parser):
             "--neuron.training_examples_per_miner",
             type=int,
             help="The number of rows to train on per miner",
-            default=500,
+            default=565,
```

Comment:
@mikkelmedm 565 here since the ratio of dataset rows to samples is roughly 2.2. 256/2.2 = 256. We could up to 512 or 1024 if we wanted to but we would have to increase the `self.train_timeout` variable on the validator side to be longer than the current 100 seconds. Increasing this even further also increases the likelihood that miners might be stuck responding to a training synapse call when the important all_reduce call comes in.